src/Requests/PostSubscriptionRequest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\NotificationClient\Requests;
+
+use JsonSerializable;
+use Keboola\NotificationClient\Exception\ClientException;
+use Keboola\NotificationClient\Requests\PostSubscription\EmailRecipient;
+use Keboola\NotificationClient\Requests\PostSubscription\Filter;
+
+class PostSubscriptionRequest implements JsonSerializable
+{
+    private const VALID_EVENT_TYPES = ['job_failed'];
+
+    protected string $eventType;
```

Comment:
proc `protected`? nic z toho nededi (tohle bude i v dalsich tridach)

Reply:
no to ses mě mel zeptat na ofsajtu po 6 pivech, to bych ti asi dal rozumnou odpověď, teď nemám tušení